### ISSUE TITLE
Add reveal password button to details pane

### ISSUE THREAD

**Issue by ahtoagah** [keepassxreboot/keepassxc#2166]
BUG: I have checked and unchecked (with keepassxc restarts) the Preference >> Security setting "Show passwords in cleartext by default", but nothing seems to change: The passwords are always hidden (black dots) in the overall database view, and the passwords are always shown in the preview pane underneath. I would like to show the passwords in the main database view

SUGGESTION: I suggest there be a global check box to ALWAYS show passwords in the window (both in the database view AND the preview pane) or NEVER. To have passwords visible in one place but not in another in the same window seems strange to me.

Thanks,
Mike

KeePassXC - Version 2.3.3
Revision: 0a155d8

Libraries:
- Qt 5.9.3
- libgcrypt 1.8.2

Operating system: macOS High Sierra (10.13)
CPU architecture: x86_64
Kernel: darwin 17.7.0

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by redm123** [comment]
Just came across this option. Doesn't seem to have any effect for me either.

**Comment by droidmonkey** [comment]
This option is only meant for when you open the entry itself. We DO NOT show passwords in the entry list or details panel for security reasons even when this setting is enabled. I do agree with adding an "eye" icon next to the shielded password in the details pane for easy revealing.

**Comment by redm123** [comment]
Both wordings make things clearer 👍

**Comment by et304383** [comment]
This is absolutely required.  I want to be able to glance at my passwords and sort them to ensure I don't have any duplicates.  Why would you omit this functionality that's available in KeePass?

**Comment by weslly** [comment]
>I thought of a nice elegant solution to this, when you right click the column bar there will be an option to reveal all passwords.

@droidmonkey don't we have this option already?

![image](https://user-images.githubusercontent.com/107672/45909253-24ee1080-bdd7-11e8-977e-f22490e3e9a5.png)

**Comment by droidmonkey** [comment]
LOL, why yes we do! Looks like I have a time machine too ;-)
